### ISSUE TITLE
Integration fails to set up device if an incorrect local IP is synced to the cloud server

### ISSUE THREAD

**Issue by GuySie** [GuySie/ha-meural#26]
The Meural Canvas syncs to the cloud server irregularly. If network details like the local IP change, it can take hours or days for this information to be updated on the server.

When Home Assistant starts up, this integration gets the local IP from the cloud server. If this IP is incorrect, the integration will fail to set up a Canvas device. This can only be resolved by restarting Home Assistant after the correct local Canvas IP has been synced to the cloud server.

**Comment by GuySie** [comment]
There's really no way to resolve this from our side - if the IP changed, there is no way to discover the Meural locally. Best solution seems to be to power cycle the Meural to force it to sync with their cloud API.

We should add a logging message to indicate this is the reason why the integration won't complete setup.

**Comment by GuySie** [comment]
Should the integration complete setup when it can't actually reach the device, hoping it will become available later? Or is it fine that this is a fatal exception that stops the integration in its tracks.

**Comment by GuySie** [comment]
If we can not reach the local IP of the canvas, we abort setup and stop updating the entity.

Ugly kludge for this now in: https://github.com/GuySie/ha-meural/commit/5af3b29ab1db29a08901aa008ce1b14a9c4a1068

**Comment by Ducky6944** [comment]
Just in case anyone runs into what I did. 

For me cycling the device did not seem to sync it to the cloud. I had updated the wifi ~3 days prior to setting up this integration. The last time it had synced was ~3 months ago. After unplugging it for ~1 hour and turning it back on the device still did not sync with the cloud. I was able to resolve this however by going to https://my.meural.netgear.com/ and uploading a photo. Immediately it synced. I disabled and re-enabled the integration and it now works as expected.
